extensions/wikia/DesignSystem/bower.json

```diff
@@ -3,6 +3,6 @@
   "private": true,
 
   "dependencies": {
-    "design-system": "Wikia/design-system#0.5.0"
+    "design-system": "Wikia/design-system#acbed9a5ecbf6b4b17502bafc9ec0047d9c4857c"
```

Comment:
We should change to the appropriate release tag it after https://github.com/Wikia/design-system/pull/52 is merged and released.